src/extension.ts

```diff
@@ -107,7 +110,28 @@ export async function activate(context: vscode.ExtensionContext) {
         return;
       }
 
-      configManager.setUserForHost(node.HostName, username);
+      configManager.setForHost(node.HostName, 'user', username);
```

Comment:
nit: i think someone can easily misspell `'user'` and it ends up saving to a different path in the config, and the bug might be hard to find because of a subtle typo. 

Would it be safer to instead just create two methods: `setUserForHost` and `setRootDirForHost` ? This way, it's much harder to make a type safety mistake?

Reply:
It is type-safe, based on the Host interface. https://github.com/tailscale-dev/vscode-tailscale/pull/119/files#diff-54ae1ba575d590a3e7d72c2b269630fb37e0562844e927e59c9ef72d39358e95R46